storage/InsertDestination.cpp

```diff
@@ -789,7 +789,7 @@ void PartitionAwareInsertDestination::returnBlockInPartition(MutableBlockReferen
                                                        << "invalidated one or more IndexSubBlocks.");
   }
   // Note that the block will only be sent if it's full (true).
-  sendBlockFilledMessage(block->getID());
+  sendBlockFilledMessage(block->getID(), part_id);
```

Comment:
In BlockPool implementation of InsertDestination, there's no information about partition ID. If that class sends a block filled message, which further gets passed to a consumer relational operator, do we use 0 as the partition ID?